js/test/Exchange/test.fetchTradingFee.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const testTradingFee = require ('./test.tradingFee.js')
+
+module.exports = async (exchange, symbol) => {
+    const skippedExchanges = []
+    if (skippedExchanges.includes (exchange.id)) {
+        console.log (exchange.id, 'found in ignored exchanges, skipping fetchTradingFee...')
+        return
+    }
+    if (exchange.has.fetchTradingFee) {
+        const fee = await exchange.fetchTradingFee ()
```

Comment:
We can't test this method without specifying the `symbol` argument which is required in this case.